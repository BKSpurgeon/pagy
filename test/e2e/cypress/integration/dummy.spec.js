/// <reference types="cypress" />

describe('Dummy Test', () => {
  it('toMatchSnapshot - HTML', () => {
    cy.visit('/')
      .then(() => {
        cy.get('#site-map').toMatchSnapshot();
      });
  });
})

