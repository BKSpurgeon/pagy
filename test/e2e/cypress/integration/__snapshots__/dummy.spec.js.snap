exports[`Dummy Test > toMatchSnapshot - HTML #0`] = `
<div id="site-map">
  <a href="/home">home</a> <a href="/bootstrap">bootstrap</a>
  <a href="/bulma">bulma</a> <a href="/foundation">foundation</a>
  <a href="/materialize">materialize</a> <a href="/navs">navs</a>
  <a href="/semantic">semantic</a> <a href="/uikit">uikit</a>
</div>
`;
